### ISSUE TITLE
Rack v1: Invalid tags in plugin.json

### ISSUE THREAD

**Issue by cschol** [stoermelder/vcvrack-packone#2]
The manifest validation script found the following issues in `plugin.json`:
```
Validating Stoermelder-P1
CVMap: invalid tags: CV, Parameter, Modulation, Map
CVPam: invalid tags: CV, Parameter, Modulation, Map
RotorA: invalid tags: Modulation
```

Valid tags are defined in:
https://github.com/VCVRack/Rack/blob/v1/src/plugin.cpp#L534

**Comment by stoermelder** [comment]
Already fixed in da7dc2f5b556abf73b4ebfe8e11cd17dfc4469ad. Will be ok in next version 1.0.2

**Comment by cschol** [comment]
Thank you!
